sdk/sdktypes/session.go

```diff
@@ -33,10 +33,8 @@ func (SessionTraits) Validate(m *SessionPB) error {
 
 func (SessionTraits) StrictValidate(m *SessionPB) error {
 	return errors.Join(
-		mandatory("created_at", m.CreatedAt),
```

Comment:
could u plz explain why created_at and state are removed from StrictValidate?

Reply:
i think it's not a must have in order, for example, to store in the db.